### PR TITLE
Misc doc changes

### PR DESCRIPTION
**Pull request by kianmeng**
Besides other changes, this commit ensures the generated HTML doc for
HexDocs.pm will become the main doc source for this Elixir library.

List of changes:
* Fix typos
* Merge all pages into readme
* Add copyright section
* Update gitignore
* Badges and more badges!
* Fix markdown

Screenshot:

Screenshot-20210218213545-2664x1464

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,24 +1,33 @@
...
+crontab-*.tar
+
+# Temporary files for e.g. tests.
+/tmp
+
 # Ignore Mix Lock since this is a lib
 /mix.lock
 
-# Dialyzer
+# Dialyzer.
```
Comment (maennchen) [inline .gitignore:31]: This is not a sentence and does therefore not need a `.`.

File: .gitignore [hunk 2]
```diff
@@ -1,24 +1,33 @@
...
 erl_crash.dump
 
 # Also ignore archive artifacts (built via "mix archive.build").
 *.ez
 
+# Ignore package tarball (built via "mix hex.build").
+crontab-*.tar
+
+# Temporary files for e.g. tests.
+/tmp
```
Comment (maennchen) [inline .gitignore:26]: The library does not use `tmp` as far as I know.

**Review by maennchen — CHANGES_REQUESTED** [review]
Thanks a lot!

I have added some minor nitpicks.

File: .gitignore (hunk 2 above)
Comment (kianmeng) [reply .gitignore:26]: > The library does not use `tmp` as far as I know.

Generated by the most recent `mix new` command, no harm keeping it there.
Let me know.

Comment (maennchen) [reply .gitignore:26]: Then we should also either add a trailing slash to it (as all the other dirs) or remove it from all dirs.

Comment (kianmeng) [reply .gitignore]: > Then we should also either add a trailing slash to it (as all the other dirs) or remove it from all dirs.

Done.

**Review by maennchen — APPROVED** [review]

**Pull request merged**